src/func/Microsoft.Health.Fhir.Ingest.Host/Microsoft.Health.Fhir.Ingest.Host.csproj

```diff
@@ -13,6 +13,7 @@
     <WarningsAsErrors />
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.1.0" />
```

Comment:
><PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.1.0" /> [](start = 4, length = 82)

So you need this new package reference, but didn't actually change any code? Would have thought a code change would go along with this.

Reply:
The change is in the connection string.  The new package allows the connection string using managed identity to function iirc.